Date: Tue, 05 Nov 1996 00:26:13 GMT
Server: NCSA/1.5
Content-type: text/html
Last-modified: Fri, 08 Sep 1995 19:34:50 GMT
Content-length: 7381

<HTML>
<HEAD>
<TITLE> Computational Biology in the UW-Madison CS Dept.</TITLE>
</HEAD>

<BODY>
<H1> <!WA0><IMG ALIGN=MIDDLE SRC="http://www.cs.wisc.edu/~shavlik//~shavlik/images/DNA-horiz.gif"> </H1>
<H1> Computational Biology in the UW-Madison CS Dept</H1>

<!WA1><img src="http://www.cs.wisc.edu/~shavlik//~shavlik/images/rainline.gif">
<P>

As a young science, computational biology offers a wealth of research
opportunities.  At the University of Wisconsin-Madison, scientists from
chemistry, computer science, genetics, mathematics, molecular biology, plant
pathology, and other disciplines are applying computational methods to various
biological problems.  Some investigations in the Department of Computer
Sciences involve DNA sequencing and analysis, experiment management, modelling
of ecological communities, protein-folding prediction, and species
identification.  Cross-disciplinary training programs are available for
graduate students interested in careers in computational biology.
<P>

Research groups investigating computational problems in biology are in three
subfields of computer science: artificial intelligence, databases, and theory.
Each group collaborates with various biological laboratories on campus.  A
summary of some of the research activities follows.
<P>

The artificial intelligence group working with 
Professor <!WA2><A HREF="http://www.cs.wisc.edu/~shavlik//~shavlik/shavlik.html">Jude Shavlik</A> applies
machine learning techniques to several problems in molecular biology.
Problems under investigation include: predicting protein secondary-structure;
distinguishing protein-coding and noncoding regions; and recognizing
promoters, splice junctions, terminators, introns, and ribosome-binding sites.
Machine learning methods aid in the discovery of concepts underlying phenomena
through the examination of multiple examples. For instance, a system can learn
to find genes by examining many DNA sequences, each classified as to whether
or not it contains a gene.  This technique is powerful and potentially very
valuable to the biological community.  Currently, the primary research focus
involves the incorporation of existing biological knowledge with computational
discovery methods.
<P>

The database group headed by 
Professors <!WA3><A HREF="http://www.cs.wisc.edu/~shavlik//~yannis/yannis.html">Yannis Ioannidis</A> 
and <!WA4><A HREF="http://www.cs.wisc.edu/~shavlik//~pubs/faculty-info/livny.html">Miron Livny</A> is
developing a desktop experiment management system that will assist scientists
in managing their experimental studies.  The goal of the system is to have a
single tool controlling the experimentation processes, managing the generated
data, and efficiently processing user requests for data.  An object-oriented
database system is at the core of the system under development.  This project
proceeds in collaboration with several laboratories on campus, primarily those
in the Departments of Soil Sciences, Molecular Biology, and Genetics.  These
groups are involved in simulation-based modelling of plant growth, microscopic
imaging, and DNA sequencing, respectively.
<P>

The research group led by 
Professor <!WA5><A HREF="http://www.cs.wisc.edu/~shavlik//~pubs/faculty-info/joseph.html">Deborah Joseph</A> 
applies techniques from
theoretical computer science to develop algorithms for computational biology
applications.  One research project in collaboration with the Wisconsin
E. coli Genome Project is leading to computational methods that generate
accurate alignments of overlapping DNA sequences.  Other work analyzes
sequence data for interesting biological features.  For instance, in
collaboration with a group in plant pathology, one class of DNA sequence is
being used to develop quantitative methods for identifying possible biological
control organisms in ecological communities.  Some of the algorithms developed
by this project have been implemented on the department's parallel computers.
<P>

Two training programs offer cross-disciplinary training for doctoral students
early in their graduate programs.  The NIH-funded Biotechnology Training
Program was established to train scientists and engineers to effectively apply
interdisciplinary research tools to solve problems of biotechnological
significance.  This program has students throughout the biological and
physical sciences involved in research problems of specific biotechnological
relevance.  The Applied Mathematics Training Program is being established to
train scientists to effectively apply mathematical and computational tools to
a wide range of scientific endeavors.  Although specific to mathematical
applications, students in this program can address a broad range of research
problems including many in the biological sciences.  Traineeships in both
programs can be awarded to students entering graduate school.<P>

<!WA6><img src="http://www.cs.wisc.edu/~shavlik//~shavlik/images/rainline.gif">
<P>

<H3> Training Programs </H3> 

<UL>
  <LI> <!WA7><A HREF="http://www.cs.wisc.edu/~shavlik//~shavlik/apply-math.html">Applied Mathematics Training Program</A>
  <LI> <!WA8><A HREF="http://www.cs.wisc.edu/~shavlik//~shavlik/biotech.html">Biotechnology Training Program</A>
</UL>

<P>
Amy Kryder (kryder@cs.wisc.edu) 
and <!WA9><A HREF="http://www.cs.wisc.edu/~shavlik//~allex/allex.html">Carolyn Allex</A> (allex@cs.wisc.edu)
are current holders of Biotechnology Training fellowships; feel free
to contact them with questions about the program.

<H3> <!WA10><A HREF="gopher://fyvie.cs.wisc.edu/11/uwcs/grad/">
     Graduate Study in Computer Sciences</A> </H3>

<H3> Electronic Access to Wisconsin Papers </H3> 
<P>
The <!WA11><A HREF="http://www.cs.wisc.edu/">Wisconsin CS department</A>
maintains an <!WA12><A HREF="ftp://ftp.cs.wisc.edu">electronic (ftp) archive</A> 
of technical reports, other papers, and software. 
<P> 
The subdirectory 
<!WA13><A HREF="ftp://ftp.cs.wisc.edu/machine-learning/shavlik-group/">
machine-learning/shavlik-group</A> contains additional papers by
Shavlik's research group.  See the file 
<!WA14><A HREF="ftp://ftp.cs.wisc.edu/machine-learning/shavlik-group/abstracts">
abstracts</A> for a list of papers, which are in compressed
postscript.  (The papers in the files 
<!WA15><A HREF="ftp://ftp.cs.wisc.edu/machine-learning/shavlik-group/shavlik.tr92.ps">
shavlik.tr92.ps</A> and 
<!WA16><A HREF="ftp://ftp.cs.wisc.edu/machine-learning/shavlik-group/craven.mlrgwp93.ps">
craven.mlrgwp93.ps</A> are recommended as the first ones to read.)

<H3> Some Interesting Links </H3>
 <UL>
  <LI> Local Links
       <UL>
  	<LI> <!WA17><A HREF="http://www.cs.wisc.edu/~shavlik//~shavlik/uwai.html"> 
       	      U-Wisc AI Group Home Page</A>
	<LI> <!WA18><A HREF="http://www.cs.wisc.edu/~shavlik//~hellers/dbmshome.html"> 
       	      U-Wisc DB Group Home Page</A>
  	<LI> <!WA19><A HREF="http://www.cs.wisc.edu/"> 
       	      U-Wisc CS Dept Home Page</A>
	<LI> <!WA20><A HREF="gopher://gopher.cs.wisc.edu"> 
              U-Wisc CS Gopher</A>
  	<LI> <!WA21><A HREF="gopher://cms.wisc.edu"> 
              U-Wisc Center for Mathematical Sciences Gopher</A>
       </UL>
  <LI> External Compbio-Related Links
       <UL>
        <LI> <!WA22><A HREF="http://golgi.harvard.edu/biopages.html">
	      Info on Biosciences</A>
	<LI> <!WA23><A HREF="http://www.gdb.org/hopkins.html">
	      Johns Hopkins Bio-Informatics Home Page</A>
        <LI> <!WA24><A HREF="http://ibc.wustl.edu/compbio">
              Wash. U. in St. Louis Inst. for Biocomputing</A>
	<LI> <!WA25><A HREF="http://www.msrc.pnl.gov:2080/docs/cie/neural/neural.homepage.html">
	      Info and Refs on Neural Nets and Molecular Science</A>
	<LI> <!WA26><A HREF="http://www.ncbi.nlm.nih.gov/">
	      Genbank</A>
       </UL>
 </UL>

<HR>
<ADDRESS> Last Changed: February 22, 1995 by shavlik@cs.wisc.edu </ADDRESS>
<HR>

</BODY>
</HTML>
